src/utils.py

```diff
@@ -42,7 +101,36 @@ def get_body(url: str, random_seed: float = None) -> str:
     WHILE_LIMIT = 4
     while_counter = 0
     headers = {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:76.0) Gecko/20100101 Firefox/76.0"}
-    cookies = {"__cfduid":"d0fe3a07bf3bfghfhstd2b0e65f076a5cc276drty7e421591486614","consentId":"dcd89b95-56ayy5esf-441b-bba7-acffab4cfd15","custom_timeout":"","euconsent":"BO0mZ6uuXwO0mZYIAAAAADM-AAAAv57_______9______9uz_Ov_v_f__33e8__9v_l_7_-___u_-23d4u_1vfu6d99yfm1-7etr3tp_87ues2_Xur__71__3z3_9pxP79r7335Ew_v-_v-b7BCPN9Y3v-8K94A","gdpr":"consented","m2_aheight":"1040","m2_analytics":"enabled","m2_awidth":"1920","m2_bot_model":"1","m2_bot_percent":"0","m2_bot_reason":"lnb","m2_click":"1","m2_height":"1080","m2_ip":"77.100.9.248","m2_keypress":"0","m2_last_unload":"22070","m2_mouse_move":"169","m2_quick_check":"true","m2_scroll":"0","m2_touch_move":"0","m2_touch_start":"0","m2_ua":"Mozilla/5.0 (Windows NT 10.0 Win64 x64 rv:76.0) Gecko/20100101 Firefox/76.0","m2_width":"1920","mm2_cookieA":"37485295-58d7-4b9f-9374-b9682949d071","pg_tc":"sample","pg_tc_response_time":"2599","PHPSESSID":"87a77b00bef041126631r47w457w58d7720aa","pv_time-1":"22070","session_depth":"1","sessionId":"e0cb9e90b-70b3c98f5f3f","tzSeconds":"3600"}
+    cookies = {"__cfduid": "d0fe3a07bf3bffghfhstd2b0e65f076a5cc276drty7e421591486614",
+               "consentId": "dcd89b95-56ayy5esf-441b-bba7-acffab4cfd15", "custom_timeout": "",
+               "euconsent": "BO0mZ6uuXwO0mZYIAAAAAADM-AAAAv57_______9______9uz_Ov_v_f__33e8__9v_l_7_-___u_-23d4u_1vfu6d99yfm1-7etr3tp_87ues2_Xur__71__3z3_9pxP79r7335Ew_v-_v-b7BCPN9Y3v-8K94A",
+               "gdpr": "consented", "m2_aheight": "1040", "m2_analytics": "enabled", "m2_awidth": "1920",
+               "m2_bot_model": "1", "m2_bot_percent": "0", "m2_bot_reason": "lnb", "m2_click": "1", "m2_height": "1080",
+               "m2_ip": "77.10.9.248", "m2_keypress": "0", "m2_last_unload": "22070", "m2_mouse_move": "169",
+               "m2_quick_check": "true", "m2_scroll": "0", "m2_touch_move": "0", "m2_touch_start": "0",
+               "m2_ua": "Mozilla/5.0 (Windows NT 10.0 Win64 x64 rv:76.0) Gecko/20100101 Firefox/76.0",
+               "m2_width": "1920", "mm2_cookieA": "374835295-58d7-4b9f-9374-b9682949d071", "pg_tc": "sample",
+               "pg_tc_response_time": "2599", "PHPSESSID": "87a77b006bef041126631r47w457w58d7720aa",
+               "pv_time-1": "22070", "session_depth": "1", "sessionId": "e0cb9e90b-70b3c98f5f3f", "tzSeconds": "3600"}
+
+    ######################################
+    #         perform url checks         #
+    ######################################
+
+    # check for dangerous url
+    if not url_safety_check(url):
+        raise UrlException(url, " Sorry, we do not currently support this new site.")
```

Comment:
*new* site? or *news* site?

As a message which is viewable to the user, I guess we should avoid typos! :-)

Reply:
lol ironic given we're a language organisation that I made a grammatical error. I'll update.